src/commands/help.js

```diff
@@ -1,44 +1,49 @@
 const Discord = require('discord.js');
+const fs = require('fs');
+const fsPromises = fs.promises;
 module.exports = {
   prefix: '!help',
+  description: 'Get the commands currently available with this bot',
   /**
    * @name help
    * Displays currently available commands.
    *
    * @param {Discord.Message} message the message provided
-   *Use this format to add new commands:
-   *{name: "Command Name", value: "Command Description"},
+   * Now automatically adds new commands - make sure the .js file has a prefix and description,
    */
 
   command: async function help(message) {
     //console.log("help", message);
     try {
-      const helpEmbed = new Discord.MessageEmbed()
-        .setColor('#0099ff')
-        .setTitle('Bot Information')
-        .setDescription(
-          'Hello! I am a test bot created by bradtaniguchi and members of FreeCodeCamp.org ' +
-            'to experiment with the process of building a Discord Bot. You can view my source ' +
-            'code at https://github.com/bradtaniguchi/discord-bot-test'
-        )
-        .addFields(
-          { name: '\u200B', value: '\u200B' },
-          {
-            name: 'Commands',
-            value: 'Here are my currently available commands!'
-          },
-          {
-            name: '!eightball <optional string>',
-            value: 'Returns a response from a magic pool ball!'
-          },
-          //new command lines go below here.
+      const helpEmbed = {
+        color: '#0099FF',
+        title: 'Bot Information',
+        description:
+          'Hello! I am a test bot created by bradtaniguchi and members of FreeCodeCamp.org to experiment with the process of building a Discord Bot. You can view my source code at https://github.com/bradtaniguchi/discord-bot-test',
+        fields: [],
+        footer: { text: 'I am not affiliated with FreeCodeCamp in any way.' }
+      };
 
-          //for spacing, leave this one at bottom.
-          { name: '\u200B', value: '\u200B' }
-        )
-        .setFooter('I am not affiliated with FreeCodeCamp in any way.');
+      fsPromises
+        .readdir(__dirname)
+        .then((result) => {
```

Comment:
Change to `async/await`, we are trying not to use `.then` and `.catch` when possible.
Also change `result` to `results`, as its an array, and thus should be named plural.

Reply:
Think I got what you were saying.